### PR TITLE
Readme: fix the explanation of how it performs updatehub-ce-server

### PR DESCRIPTION
**Pull request by chtavares592**
Signed-off-by: Christian Tavares <christian.tavares@ossystems.com.br>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ docker run -d -p 8080:8080 updatehub/updatehub-ce-server:latest
 
 ## Development setup
 
-The `updatehub-ce-server` uses `glide` to manage its dependencies. The
-easiest way to install its latest release on Mac or Linux is with the
+The `updatehub-ce-server` uses `glide` to manage its dependencies and `npm` to build the interface
+on the server. The easiest way to install its latest release on Mac or Linux is with the
```
Comment (otavio) [inline README.md]: ... to build the web UI for the server.

File: README.md [hunk 2]
```diff
@@ -38,23 +38,47 @@ $ curl https://glide.sh/get | sh
...
+
+```
+$ go get -u github.com/gobuffalo/packr/packr
+```
+
+To get the dependency
+
+```
+$ go get -u github.com/gobuffalo/packr
+```
```
Comment (otavio) [inline README.md]: Is this need? I think the first does it as well.

File: README.md [hunk 3]
```diff
@@ -38,23 +38,47 @@ $ curl https://glide.sh/get | sh
...
+
+
+Finally, you can build `updatehub-ce-server` as:
 
 ```
+$ cd <YOUR-UPDATEHUB-CE-SERVER-PATH>/ui/ && sudo npm install && npm run build && cd ..
```
Comment (otavio) [inline README.md]: Does it need to be run as root?

File: README.md (hunk 2 above)
Comment (chtavares592) [reply README.md]: Yes, need to install the booth

File: README.md (hunk 3 above)
Comment (chtavares592) [reply README.md]: No, it doesn't .. I already removed

**Pull request merged**